sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -53,6 +54,16 @@ public void serialize(DiagnosticsClientConfig clientConfig, JsonGenerator genera
                 generator.writeStringField("machineId", ClientTelemetry.getMachineId(clientConfig));
                 generator.writeStringField("connectionMode", clientConfig.getConnectionMode().toString());
                 generator.writeNumberField("numberOfClients", clientConfig.getActiveClientsCount());
+                generator.writeObjectFieldStart("clientEndpoints");
+                for (String key: clientConfig.clientMap.keySet()) {
```

Comment:
why not directly use the map format:
ClientEndpoints:
{
"databaseAccount1": 1,
"databaseAccount2": 2
}

Reply:
agreed, that looks much better, thanks!